dss-policy-jaxb/src/main/resources/xsd/policy.xsd

```diff
@@ -141,6 +141,16 @@
 		</xs:sequence>
 	</xs:complexType>
 	
+	<xs:complexType name="CACertificateConstraints">
+		<xs:complexContent>
+			<xs:extension base="tns:CertificateConstraints">
+				<xs:sequence>
+					<xs:element type="xs:base64Binary" name="TrustPoints" maxOccurs="unbounded" minOccurs="0" />
```

Comment:
contains X509Certificate anchors encoded (probably better naming is required here, I followed the old standard for signature policies RFC 3125, ETSI TR 102 272 and ETSI TR 102.038)